rotkehlchen/accounting/constants.py

```diff
@@ -1,206 +1,264 @@
+        HistoryEventSubType.RETURN_WRAPPED: EventCategory.SEND,
+        HistoryEventSubType.LIQUIDATE: EventCategory.LIQUIDATE,
+        HistoryEventSubType.PAYBACK_DEBT: EventCategory.REPAY,
         HistoryEventSubType.FEE: EventCategory.FEE,
+        HistoryEventSubType.DONATE: EventCategory.DONATE,
+        HistoryEventSubType.DEPOSIT_ASSET: EventCategory.DEPOSIT,
+        HistoryEventSubType.NONE: EventCategory.SEND,  # create for basic decoded events
     },
     HistoryEventType.WITHDRAWAL: {
-        HistoryEventSubType.NONE: EventCategory.WITHDRAW,
         HistoryEventSubType.REMOVE_ASSET: EventCategory.WITHDRAW,
-        HistoryEventSubType.GENERATE_DEBT: EventCategory.BORROW,
+        HistoryEventSubType.NONE: EventCategory.WITHDRAW,
         HistoryEventSubType.BRIDGE: EventCategory.BRIDGE_OUT,
+        HistoryEventSubType.CANCEL_ORDER: EventCategory.CANCEL_ORDER,
+        HistoryEventSubType.REFUND: EventCategory.REFUND,
+        HistoryEventSubType.GENERATE_DEBT: EventCategory.BORROW,
+        HistoryEventSubType.RETURN_WRAPPED: EventCategory.SEND,
+        HistoryEventSubType.FEE: EventCategory.FEE,
+        HistoryEventSubType.REWARD: EventCategory.CLAIM_REWARD,  # verify
     },
-    HistoryEventType.DEPOSIT: {
-        HistoryEventSubType.NONE: EventCategory.DEPOSIT,
-        HistoryEventSubType.DEPOSIT_ASSET: EventCategory.DEPOSIT,
-        HistoryEventSubType.BRIDGE: EventCategory.BRIDGE_IN,
-        HistoryEventSubType.PLACE_ORDER: EventCategory.PLACE_ORDER,
+    HistoryEventType.TRADE: {
+        HistoryEventSubType.SPEND: EventCategory.SWAP_OUT,
+        HistoryEventSubType.RECEIVE: EventCategory.SWAP_IN,
+        HistoryEventSubType.NONE: EventCategory.INFORMATIONAL,  # verify
+        HistoryEventSubType.FEE: EventCategory.FEE,  # used in exchanges
     },
-    HistoryEventType.MIGRATE: {
-        HistoryEventSubType.SPEND: EventCategory.SEND,
-        HistoryEventSubType.RECEIVE: EventCategory.RECEIVE,
+    HistoryEventType.RENEW: {
+        HistoryEventSubType.NFT: EventCategory.RENEW,
     },
     HistoryEventType.STAKING: {
         HistoryEventSubType.DEPOSIT_ASSET: EventCategory.DEPOSIT,
-        HistoryEventSubType.RECEIVE_WRAPPED: EventCategory.RECEIVE,
+        HistoryEventSubType.REWARD: EventCategory.STAKING_REWARD,
         HistoryEventSubType.REMOVE_ASSET: EventCategory.WITHDRAW,
-        HistoryEventSubType.RETURN_WRAPPED: EventCategory.SEND,
-        HistoryEventSubType.MEV_REWARD: EventCategory.MEV_REWARD,
         HistoryEventSubType.BLOCK_PRODUCTION: EventCategory.CREATE_BLOCK,
-        HistoryEventSubType.REWARD: EventCategory.STAKING_REWARD,
+        HistoryEventSubType.MEV_REWARD: EventCategory.MEV_REWARD,
+    },
+    HistoryEventType.TRANSFER: {
+        HistoryEventSubType.DONATE: EventCategory.DONATE,
+        HistoryEventSubType.NONE: EventCategory.TRANSFER,
+    },
+    HistoryEventType.UNKNOWN: {
+        HistoryEventSubType.NONE: EventCategory.INFORMATIONAL,
+    },
```

Comment:
Do we need this? Where is this type used? I don't see this type used anywhere. Shouldn't we remove the type itself and the mapping?

Reply:
This one comes from the default value: https://github.com/rotki/rotki/blob/develop/rotkehlchen/tests/utils/factories.py#L131
The tests that access it are these:
```
rotkehlchen/tests/db/test_history_events.py::test_delete_last_event
rotkehlchen/tests/unit/decoders/test_main.py::test_maybe_reshuffle_events
rotkehlchen/tests/api/test_ethereum_transactions.py::test_events_filter_params
rotkehlchen/tests/api/test_ethereum_transactions.py::test_ignored_assets
```
I'll change it to something common like TRANSFER/NONE, so UNKNOWN could be removed completely.